source/site_ops/how-tos/enable_public_course_content.rst

```diff
@@ -0,0 +1,120 @@
+.. _Enable Public Course Content:
+
+####################################
+Enabling Public Course Content
+####################################
+
+By default, learners must create an Open edX account, be signed in to the LMS, 
```

Comment:
```suggestion
By default, learners must create an account on your Open edX instance, be signed in to the LMS, 
```

Not sure if this is the right phrasing but "Open edX account" might be misconstrued as an account on the Open edX sandbox or on openedx.org